### PR TITLE
Chapter 1 / Homework 1And2

### PR DESCRIPTION
**Pull request by dvjstefan**
My first PR... Did i make it or not?! :o

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -7,6 +7,8 @@ class Program
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello World!");
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Program.cs]: This is not what the solution of homework 2 looks like. Please take a look at the README.md file of that branch or read the homework description in Discord again! :)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -7,6 +7,39 @@ class Program
...
+
+            Program.HomeworkOne();
+            //Program.HomeworkOneB("Stefan", "Andersson", 41f, 95f, 178f);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5069627)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5069627&bid=16680794)

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (dvjstefan) [reply Src/BootCamp.Chapter/Program.cs]: Better now..? *cross my fingers, toes, arms, head, body... everything*

Comment (Almantask) [reply Src/BootCamp.Chapter/Program.cs]: Hello world is not needed. Please do take a look at the readme and ask us questions if it's still not clear :)

**Pull request closed without merging**